src/frontend/test-setup.ts

```diff
@@ -12,6 +19,19 @@ declare global {
         isUserConsenting: boolean
       ) => Promise<string>;
       removeVirtualWebAuth: (authenticatorId: string) => Promise<void>;
+      getWebauthnCredentials: (
+        authenticatorId: string
+      ) => Promise<WebAuthnCredential[]>;
+      addWebauthnCredential: (
+        authenticatorId: string,
+        rpId: string,
+        credentialId: string,
+        isResidentCredential: boolean,
+        privateKey: string,
+        signCount: number,
+        userHandle?: string,
+        largeBlob?: string
+      ) => Promise<void>;
```

Comment:
```suggestion
      addWebauthnCredential: (WebAuthnCredential & { rpId: string, userHandle?: string, largeBlob?: string, authenticatorId: string }) => Promise<void>;
```

Reply:
This does not work, as the function takes separate arguments only and not an object.